grammar.js

```diff
@@ -648,6 +676,35 @@ module.exports = grammar({
       $._statement,
     ),
 
+    _top_level_expression_statement: $ => seq(
+      optional(choice(
```

Comment:
same, it's not clear what is the difference with the general expression.

Reply:
updated